lib/flatpak-installation.c

```diff
@@ -1168,26 +1174,31 @@ progress_cb (OstreeAsyncProgress *progress, gpointer user_data)
                                   formatted_bytes_sec, formatted_bytes_transferred,
                                   formatted_total);
 
-          new_progress = (100 * bytes_transferred) / total_delta_part_size;
+          fetched_size += bytes_transferred;
+          total_size += total_delta_part_size;
```

Comment:
I don't really see how this makes a difference in behaviour, because only one of the if-else-if-.. branches will be taken, so fetched_size is only updated once.